src/pages/derivx/_accounts.tsx

```diff
@@ -1,13 +1,24 @@
-import React from 'react'
+import React, { ReactElement } from 'react'
 import styled from 'styled-components'
 import { Flex, SectionContainer } from 'components/containers'
 import { Header, Text } from 'components/elements'
 import { LinkButton } from 'components/form'
 import { Localize, localize } from 'components/localization'
-import DxFinancialIcon from 'images/svg/deriv-x/dxfinancial.svg'
 import DxSyntheticIcon from 'images/svg//deriv-x/dxsynthetic.svg'
+import DxFinancialIcon from 'images/svg/deriv-x/dxfinancial.svg'
 import device from 'themes/device'
 
+type StyledHeaderProps = {
+    'font-size': string
+    margin: string
+}
+
+type ContentType = {
+    header: ReactElement
+    text: ReactElement | string
+    icon: ReactElement | string
```

Comment:
```suggestion
    text: ReactElement | string
    icon: ReactElement | string
```
We should be strict and maintain only one type for this. `ReactElement` would be good for `text` since we would want the content to be always localized.